Library/Homebrew/cmd/list.rb

```diff
@@ -165,9 +167,9 @@ def filtered_list(args:)
     end
   end
 
-  def list_casks(args:)
+  def list_casks(casks, args)
```

Comment:
```suggestion
  def list_casks(casks, args:)
```

We use a keyword argument here elsewhere so doing so here is good for consistency